### PR TITLE
Remove patch 205 and 178

### PR DESCRIPTION
**Pull request by hroncok**
Those are fixup commits. Will rebase. The 205 (and 178) fixup removes the 205 (and 178) commit entirely.

### CODE REVIEW

**Comment by hroncok** [comment]
See the result in:

https://github.com/hroncok/cpython/commits/3.8.0a4-rebased

**Comment by hroncok** [comment]
Task info: https://koji.fedoraproject.org/koji/taskinfo?taskID=34891829

**Comment by hroncok** [comment]
here I also remove 178:
Task info: https://koji.fedoraproject.org/koji/taskinfo?taskID=34891883

File: configure.ac [hunk 1]
```diff
@@ -4653,9 +4653,9 @@ fi
 dnl define LIBPL after ABIFLAGS and LDVERSION is defined.
 AC_SUBST(PY_ENABLE_SHARED)
 if test x$PLATFORM_TRIPLET = x; then
-  LIBPL='$(prefix)'"/lib/python${VERSION}/config-${LDVERSION}"
+  LIBPL='$(prefix)'"/lib64/python${VERSION}/config-${LDVERSION}"
 else
-  LIBPL='$(prefix)'"/lib/python${VERSION}/config-${LDVERSION}-${PLATFORM_TRIPLET}"
+  LIBPL='$(prefix)'"/lib64/python${VERSION}/config-${LDVERSION}-${PLATFORM_TRIPLET}"
```
Comment (hroncok) [inline configure.ac:4658]: And in configure.ac, we can just add "64" (as we already do in plenty spaces) in patch 102.

**Comment by hroncok** [comment]
https://src.fedoraproject.org/rpms/python38/pull-request/20

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**